src/export.js

```diff
@@ -94,8 +94,8 @@ class Exporter {
                 // determine end of week () which would be saturday
                 .endOf('week')
                 // the duration must go to the start of the new week
-                // sunday + monday === 2 days
-                .add(2, 'd');
+                // which is sunday
```

Comment:
@LonnyGomes Two days were being added intentionally but that clearly doesn't get us what we want!